std_msgs/Float32MultiArray pos
---
std_msgs/Float32 rew